client/app/services/routes.js

```diff
@@ -0,0 +1,21 @@
+import { isString, isObject, filter } from "lodash";
+
+class Routes {
+  items = [];
```

Comment:
Same thing I commented in https://github.com/getredash/redash/pull/5000#discussion_r445175371 (or almost), update if you think it makes sense

Reply:
The answer is the same for both cases, so I replied in the thread you mentioned ^